src/main/java/org/folio/services/kafka/KafkaProperties.java

```diff
@@ -1,44 +1,29 @@
 package org.folio.services.kafka;
 
-import static java.lang.Integer.parseInt;
 import static java.lang.Short.parseShort;
 import static java.lang.System.getenv;
 
-import java.util.Map;
-import org.apache.kafka.common.serialization.StringSerializer;
-
 public final class KafkaProperties {
-  private static int port = parseInt(getenv().getOrDefault("KAFKA_PORT", "9092"));
+  private static String port = getenv().getOrDefault("KAFKA_PORT", "9092");
```

Comment:
Any advantage to use `string` over `int` for `port`, or just try to match with `KafkaConfig` in `folio-kafka-wrapper`?

Reply:
I tried to match the `folio-kafka-wrapper`